src/deepsparse/transformers/helpers.py

```diff
@@ -193,3 +199,93 @@ def _normalize_fields(_dict):
         return result
 
     return _wrapper
+
+
+def truncate_transformer_onnx_model(
+    model_path: str,
+    emb_extraction_layer: int = -1,
+    model_size: Optional[int] = None,
```

Comment:
will deepsparse crash without this? would prefer to remove if not as this requires extra user knowledge. if we must keep, lets rename to `hidden_layer_size` or something like that

Reply:
Run time is faster if `model_size` is already known, since it allows the the engine to preallocate space. The engine does not crash if it's not provided, that's why I made it optional for faster run time.